projects/dex-ui/src/components/Swap/SwapRoot.tsx

```diff
@@ -10,15 +10,20 @@ import { useAccount } from "wagmi";
 import { Quote, QuoteResult } from "@beanstalk/sdk/Wells";
 import { Button } from "./Button";
 import { Log } from "src/utils/logger";
+import { useParams, useSearchParams } from "react-router-dom";
 import { TransactionToast } from "../TxnToast/TransactionToast";
 
 export const SwapRoot = () => {
   const { address: account } = useAccount();
 
+  const [tokenSwapParams, setTokenSwapParams] = useSearchParams();
+  const token1 = tokenSwapParams.get("token1")
+  const token2 = tokenSwapParams.get("token2")
```

Comment:
Please rename these to `fromToken` and `toToken`

Reply:
Fixed!